src/Security/Authorization/Core/src/DefaultAuthorizationPolicyProvider.cs

```diff
@@ -71,4 +71,11 @@ public Task<AuthorizationPolicy> GetDefaultPolicyAsync()
         // A change to either of these behaviors would require shipping a patch of MVC as well.
         return Task.FromResult(_options.GetPolicy(policyName));
     }
+
+#if NETCOREAPP
+    /// <summary>
+    /// Determines if policies from this provider can be cached, which is true for this provider.
+    /// </summary>
+    public virtual bool CanCachePolicy => true;
```

Comment:
I wonder if this should be:

```suggestion
    public virtual bool CanCachePolicy => GetType() == typeof(DefaultAuthorizationPolicyProvider);
```

Reply:
Makes sense